vignettes/articles/enteric_fever_example.Rmd

```diff
@@ -136,7 +136,7 @@ We examine our cross-sectional antibody data by visualizing the distribution of
 ```{r plots}
 #Create plots
 
-xs_data %>% autoplot(strata = "Country")
+xs_data %>% autoplot(strata = "Country",type='density')
```

Comment:
Please add space after comma. I won't mark other instances of this; is there a way to check for this automatically? There's something called "linting" that I've never learned about

Reply:
This is well-noted. There's a [Lintr](https://rowannicholls.github.io/R/intro/linting.html) package that checks for adherence to a given style.